examples/src/main/java/tachyon/examples/JournalCrashTest.java

```diff
@@ -276,8 +276,8 @@ public static void main(String[] args) {
       LOG.info("Round {}: Planning Master Alive Time {}ms.", rounds, aliveTimeMs);
 
       System.out.println("Round " + rounds + " : Launch Clients...");
-      sTfs = TachyonFileSystemFactory.get();
-      sOldTfs = TachyonFS.get(ClientContext.getConf());
+      sTfs = TachyonFileSystem.TachyonFileSystemFactory.get();
```

Comment:
I think Intellij will do `TachyonFileSystem.TachyonFileSystemFactory` until you start using it multiple times to avoid the import.